typescript/token/package.json

```diff
@@ -35,7 +35,8 @@
   },
   "files": [
     "/dist",
-    "/contracts"
+    "/contracts",
+    "/docs"
```

Comment:
This would cause the docs to get included in what gets published to NPM. Is that intentional?

Reply:
yeah this is how we are pulling into the docs repo atm 
https://github.com/hyperlane-xyz/docs/blob/main/package.json
probably not the ideal solution